pollen_vision/pollen_vision/camera_wrappers/camera_wrapper.py

```diff
@@ -19,6 +19,11 @@ def get_data(
         exit()
 
     @abstractmethod
-    def get_K(self, left: bool = True) -> npt.NDArray[np.float32]:
+    def get_K(self, cam_name: str) -> npt.NDArray[np.float32]:
         self._logger.error("Abstract class CameraWrapper does not implement get_K()")
```

Comment:
actually we can replace the body of the function by a pass.
The abstractmethod decorator will raise and error if the class is instantiated

Reply:
Done